components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -478,6 +489,34 @@ && isStepHasMultiOption(context)
         }
     }
 
+    private boolean checkIfApplicationEnabled(HttpServletRequest request, AuthenticationContext context)
+            throws FrameworkException {
+
+        String type = request.getParameter(TYPE);
```

Comment:
what is the type value? is that the app protocol (oauth/ saml)

Reply:
Yes. It is the client type.